tools/domspurge/README.md

```diff
@@ -0,0 +1,37 @@
+# Purge DOMS/CDMS Execution Data
+
+## Prerequisites
+
+_If you are running the `deletebyquery.py` script from within the nexus-webapp-driver Docker image, the following prerequisites are not necessary._
```

Comment:
Do you mean running the purge.py script here?

Reply:
@ngachung 
Yes. I copy/pasted that text from deletebyquery's README and forgot to change it. 
Corrected